content/en/blog/_posts/2023-04-04-recent-developments-in-kubelet-to-speed-up-pod-startup.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: blog
+title: "Recent developments in kubelet to speed up Pod startup"
+date: 2023-04-04T00:00:00+0000
+slug: speed-up-pod-startup
+---
+
+**Authors**: Paco Xu(DaoCloud), Sergey Kanzhelev(Google), Ruiwen Zhao(Google)
+
+How can pod start-up time be accelerated on nodes in large clusters? This is a common issue that
```

Comment:
You accelerate Pod start-up, or your shorten the Pod start-up time ...

Reply:
Fixed.